test/NeoSharp.Core.Test/Network/Protocols/UtProtocolV2.cs

```diff
@@ -19,6 +20,7 @@ public class UtProtocolV2 : TestBase
         public void WarmSerializer()
         {
             AutoMockContainer.Register<IBinarySerializer>(new BinarySerializer(typeof(VersionMessage).Assembly));
+            AutoMockContainer.Register<IBinaryDeserializer>(new BinaryDeserializer(typeof(VersionMessage).Assembly));
```

Comment:
Duplication?

Reply:
If in the future, we cache it per object, maybe not